styles.css

```diff
@@ -0,0 +1,125 @@
+
+.mobile-header {
+  position: absolute;
```

Comment:
- [ ] Please avoid using the `position` property unless is necessary, everything in this portfolio project could be made using `display: flex` and `display: grid` with the default position (unmodified)